pytest.ini

```diff
@@ -0,0 +1,5 @@
+[pytest]
+testpaths = tests/
```

Comment:
having this line cuts test discovery time locally from ~10 seconds to <1sec - particularly nice when calling `pytest -k ...`

I didn't pay attention to the other lines, just copied from `dash`

Reply:
🎉